code/game/turfs/flooring/_flooring.dm

```diff
@@ -13,43 +13,53 @@ var/global/list/flooring_cache = list()
 	var/desc
 	var/icon
 	var/icon_base
-	var/color
+	var/color = COLOR_WHITE
 	var/footstep_type = /decl/footsteps/blank
+	var/growth_value = 0
+
+	var/neighbour_type
```

Comment:
do we have like. a policy on AmEng vs BrEng in variable names btw? and in strings? may be worth having a unified style when it comes to that